SPEC.md

```diff
@@ -56,11 +56,10 @@ In some ways an ACI can be thought of as equivalent to a static binary.
 ### Image Layout
 
 The on-disk layout of an App Container Image is straightforward.
-It includes a *rootfs* with all of the files that will exist in the root of the app, and an *app image manifest* describing the contents of the image and how to execute the app.
+It includes a *rootfs* directory with all of the files that will exist in the root of the app, and an *app image manifest* file describing the contents of the image and how to execute the app.
 
 ```
 /manifest
-/rootfs
```

Comment:
I'm sympathetic to the ambiguity; I think we just landed on that because
it's standard tar tf output.
We could just use a real tar tvf example to be completely clear..
Am 04.05.2015 20:15 schrieb "Tim Hockin" <notifications@github.com>:

> In SPEC.md <https://github.com/appc/spec/pull/335#discussion_r29641789>:
>
> >
> >  ```
> >  /manifest
> > -/rootfs
>
> I omitted it precisely because you are trying to have it both ways. My
> real gripe was that it was not obvious that of the two top-level items one
> is a file and the other a dir. Not too attached to the change, I guess.
> Just unclear.
> … <#14d2211f463f7217_>
> On May 4, 2015 7:50 PM, "Jonathan Boulle" <notifications@github.com>
> wrote: In SPEC.md <
> https://github.com/appc/spec/pull/335#discussion_r29641278>: > > ``` >
> /manifest > -/rootfs What's your motivation for omitting this? This
> arguably conflicts with the second stipulation under "Image Archives" (line
> 75/74). (I realise we're trying to have it both ways here - by not showing
> the rootfs/usr/ or rootfs/usr/bin/ directories - but imo seems like it's
> better to err on the side of verbosity and include them instead of removing
> this) — Reply to this email directly or view it on GitHub <
> https://github.com/appc/spec/pull/335/files#r29641278>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appc/spec/pull/335/files#r29641789>.
>